invite/settings.py

```diff
@@ -36,4 +36,4 @@ def get_service_name():
     return getattr(
         settings,
         'INVITE_SERVICE_NAME',
-        Site.objects.get_current().name)
+        Site.objects.get_current(request=request).name)
```

Comment:
When I run this in Django 1.6 and try to send an invite, I get: `TypeError at /invite/ get_current() got an unexpected keyword argument 'request'`. Also, tests are failing in some Django versions for me with similar errors. Would it work to use `django.contrib.sites.models.get_current_site()` instead of Site.objects.get_current() for compatibility (assuming others are able to replicate my issue)?